### DISCUSSION TITLE
Audio examples

### DISCUSSION THREAD

**Discussion by bkumanchik** [raysan5/raylib#1936] [Q&A] [answered]
I can't get anything to do with audio to work at all even the audio examples from the examples folder, Anyone else have audio trouble?

Brian

**Comment by bkumanchik** [comment]
Anyone?

**Comment by raysan5** [comment, marked as answer]
@bkumanchik they work for me on windows, are you using latest raylib from github master branch? which platform?

**Reply by bkumanchik** [reply]
I can't make the master build work :(

**Reply by bkumanchik** [reply]
OK, so the MinGW (which is at 3.7) works just fine but the TCC version (which is at 3.5) has audio issues.

Thanks for your help,

Brian

**Comment by bkumanchik** [comment]
No, I got it from Ich.io and I'm on a PC. I'll try GitHub
Did you just run them off the web browser 
    On Friday, August 20, 2021, 01:11:40 PM PDT, Ray ***@***.***> wrote:  
 
 


@bkumanchik they work for me on windows, are you using latest raylib from github master branch? which platform?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
